### DISCUSSION TITLE
🙏 Help: InputDate im Voice Over Screen Reader

### DISCUSSION THREAD

**Discussion by agnieszkaw61** [public-ui/kolibri#6644] [Q&A]
### Summary

Halo,
Wenn wir das Eingabedatum in Google Chrome mit dem VoiceOver-Screenreader auf dem Mac verwenden, erhalte ich so etwas:
![image](https://github.com/user-attachments/assets/75078254-5030-41c5-a707-0e8bf011823c)
Ich habe Information uber Tag / Monat, und Labelname aber warum sind diese Prozente auch gelesen? Ist das ein Bug bei dir? Im Windows Narrator gibt es nichts Vergleichbares.

**Comment by deleonio** [comment]
Hi @agnieszkaw61 , vielen Dank, wir testen das nach und suchen eine Lösung.
